src/main/java/com/google/api/codegen/discovery/transformer/php/PhpSampleMethodToViewTransformer.java

```diff
@@ -169,7 +170,14 @@ private SamplePageStreamingView createSamplePageStreamingView(
     builder.isResourceMap(fieldInfo.type().isMap());
     builder.pageVarName(
         symbolTable.getNewSymbol(namer.localVarName(Name.lowerCamel(fieldInfo.name()))));
-    builder.pageTokenName(methodInfo.requestPageTokenName());
+    boolean isResourceSetterInRequestBody = methodInfo.isPageStreamingResourceSetterInRequestBody();
+    builder.isResourceSetterInRequestBody(isResourceSetterInRequestBody);
+    // Use upper camel case for getter/setter function names in templates.
+    if (isResourceSetterInRequestBody) {
+      builder.pageTokenName(Name.lowerCamel(methodInfo.requestPageTokenName()).toUpperCamel());
```

Comment:
Use the namer for this, you can either add a `getPageTokenSetterFuncName` to the `PhpSampleNamer` or call `namer.publicMethodName(Name.lowerCamel("get", methodInfo.requestPageTokenName()))`

That way there's no name composition in the snippet